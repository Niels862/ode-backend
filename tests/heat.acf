let ClkA = 1;

let alpha = 1;
let beta = 4;

chip {
    io: [
        input,
        input,
        output
    ],
    cabs: [
        cab 1 with clocks ClkA, - {
            cams: [
                SumInv as sum1 {
                    inputs: 3,
                    gain1: alpha,
                    gain2: alpha,
                    gain3: alpha,
                }
            ]
        },
        cab 2 with clocks ClkA, - {
            cams: [
                SumInv as sum2 {
                    inputs: 3,
                    gain1: alpha,
                    gain2: alpha,
                    gain3: alpha,
                }
            ]
        },
        cab 3 with clocks ClkA, - {
            cams: [
                GainInv as gain1 {},
                GainInv as gain2 {}
            ]
        },
        cab 4 with clocks ClkA, - {
            cams: [
                Integrator as integ1 {
                    integ_const: beta,
                },
                Integrator as integ2 {
                    integ_const: beta,
                }
            ]
        }
    ],
    routing: [
        io1 -> sum1:1,
        integ1 -> sum1:2,
        gain2 -> sum1:3,
        sum1 -> integ1,
        integ1 -> gain1,

        gain1 -> sum2:1,
        integ2 -> sum2:2,
        io2 -> sum2:3,
        sum2 -> integ2,
        integ2 -> gain2,

        integ2 -> io3
    ]
}
