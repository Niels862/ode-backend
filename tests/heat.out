0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 128 0 0 0 1 0 16 0 0 0 0 0 16 0 0 64 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 26 1 129 1 29 1 129 1 25 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 160 0 16 0 0 0 208 0 16 0 5 0 144 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 19 1 129 1 25 1 129 1 30 1 129 1 29 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 144 0 16 0 0 0 224 0 16 0 5 0 208 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 18 1 130 1 30 1 130 1 19 1 129 1 31 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 32 0 32 0 224 0 32 0 5 0 48 0 16 0 5 0 240 0 16 0 0 0 0 255 255 255 255 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 32 0 32 0 48 0 16 12 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 5 1 25 1 40 0 5 1 27 1 24